packages/amplify-codegen-model-plugin/src/languages/typescript-declaration-block.ts

```diff
@@ -0,0 +1,267 @@
+import { indent, indentMultiline } from '@graphql-codegen/visitor-plugin-common';
+import { access } from 'fs';
```

Comment:
We standardized on ```fs-extra``` and not using ```fs``` anywhere

Reply:
This is the side effect of vscode being too helpful. It auto imported this, and I had wrong thing pushed into an array in https://github.com/aws-amplify/amplify-cli/pull/2756/files#diff-d8a19b27e070f63d0ceec21e7b643dceR232